frontend/src/components/admin/AddKpi.tsx

```diff
@@ -72,12 +71,12 @@ export default function AddKpi(props: AddKpiProps) {
                         <MenuItem value={"EQUALS"}>gleich</MenuItem>
                     </Select>
                 </FormControl>
-                <FormControl sx={{m: 1, minWidth: 80}}>
+                <FormControl sx={{m: 1, minWidth: 300}}>
```

Comment:
Könntest du hier ID nich einstellen? um danach in css file einmal für alle FormControl alles zu definieren.
Wenn ich mich nicht irre, versteht Mui keine ClassNames, aber mit IDs ist alles ok, ich nutze das für meine Buttons